daemon/images/image_prune.go

```diff
@@ -163,7 +163,7 @@ func imageDeleteFailed(ref string, err error) bool {
 	switch {
 	case err == nil:
 		return false
-	case errdefs.IsConflict(err):
+	case errdefs.IsConflict(err), errors.Is(err, context.Canceled):
```

Comment:
```suggestion
	case errdefs.IsConflict(err), errors.Is(err, context.Canceled), errors.Is(err, context.DeadlineExceeded):
```
Annoyingly, you need to enumerate all the possible context errors.

Reply:
LOL; I actually had the DeadlineExceeded, and started to think if that one would be one to actually _log_, so removed it; let me add it back 😂

Reply:
And it's back (it was _literally_ `CMD+Z` - last edit I made 😂 )